OFFLINE-DOCS.md

```diff
@@ -0,0 +1,12 @@
+# Offline docs instructions
```

Comment:
This file only mentions the `./Dockerfile` file. What about the contents of `offline-build`?

Reply:
I've deleted the root Dockerfile